config/kn-plugin-event.yaml

```diff
@@ -1,17 +1,17 @@
 config:
   branches:
-    release-1.13:
-      openShiftVersions:
-        - version: '4.15'
-    release-1.14:
-      openShiftVersions:
-        - version: '4.15'
     release-1.15:
       konflux:
         enabled: true
       openShiftVersions:
         - version: '4.16'
           useClusterPool: true
+    release-1.16:
+      konflux:
+        enabled: true
+      openShiftVersions:
+        - version: '4.17'
+          useClusterPool: true
```

Comment:
AFAIK, we don't have a cluster pool for 4.17 yet. The version here should be 4.16 for now. When we bump the cluster pool we can increase the version in this place and in other repository configs as well.

Reply:
I backed down it to 4.16 cluster pool in 02e43dad1961425861c9406246f1d19f2d4ee911 for now.